src/pages/household/output/PolicyImpactPopup.jsx

```diff
@@ -11,7 +11,15 @@ export default function PolicyImpactPopup(props) {
     setHasShownPopulationImpactPopup,
   } = props;
   const content = (
-    <div>
+    <div
+      style={{
+        display: "flex",
+        flexDirection: "column",
+        alignItems: "center",
+        justifyContent: "center",
+        textAlign: "center",
+      }}
+    >
       <div>
         {metadata.countryId === "us" ? (
           <p>
```

Comment:
Before testing this, just a minor request: could you refactor this so that instead of conditionally displaying two links baked into the component, the links are stored in a separate object where the key is the country ID, then we conditionally render the link address only?